packages/app/src/app/pages/Dashboard/Content/routes/Discover/contants.ts

```diff
@@ -2,8 +2,7 @@ export const FEATURED_SANDBOXES_ALBUM = 'i1pxky';
 export const TRENDING_SANDBOXES_ALBUM = 'nynbnp';
 
 export const banner = {
-  label: 'NEW FEATURE',
-  title: 'Welcome Discover Page',
+  title: 'Discover Page',
```

Comment:
Changed the title from `Welcome Discover Page` to simply `Discover Page` because the wording wasn't right.

Reply:
cc @filipelima18